twisted/conch/ssh/transport.py

```diff
@@ -1593,7 +1747,6 @@ def _continueKEXDH_REPLY(self, ignored, pubKey, f, signature):
             return
         self._keySetup(sharedSecret, exchangeHash)
 
-
```

Comment:
similarly, this removal of whitespace is both unnecessary and in violation of the coding standard :).  Twisted's vertical whitespace requirement is different than pep-8 - 3 lines separate top-level suites, 2 lines separate method-level suites, and 1 line can be used to separate sections within a method.